test/ClangDisplayContext/template.json

```diff
@@ -0,0 +1,7 @@
+[
+{
+  "directory": "test_dir",
+  "command": "clang++ -std=c++11 -o test.o test_dir/test.cpp",
```

Comment:
Do we need the output of the compilation for anything? @Xazax-hun Won't `-o/dev/null` be better?